doc/lightning-withdraw.7

```diff
@@ -37,6 +37,10 @@ the suffix is equivalent to \fIperkb\fR\.
 \fIminconf\fR specifies the minimum number of confirmations that used
 outputs should have\. Default is 1\.
 
+
+\fIutxos\fR specifies the utxos to be used to be withdrawn from, as an array
+of "txid:vout"\. These must be drawn from the node's available UTXO set\.
```

Comment:
Also here. They are the new variables in this doc.

Reply:
generally, this is a good rule. in this case, however, using a txid:vout to describe a utxo is rather canonical, and should be self-explanatory to a user who's interested in this option.